main.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+
+# Create methods that do the following:
+# List all books.
+# List all people.
+# Create a person (teacher or student, not a plain Person).
+# Create a book.
+# Create a rental.
+# List all rentals for a given person id.
+
+require './classroom'
+require './person'
+require './teacher'
+require './student'
+
+def list_books
+  books = [Book.new('Harry Potter', 'JK Rowling'),
+           Book.new('Lord of the Rings', 'JRR Tolkien'),
+           Book.new('The Hobbit', 'JRR Tolkien')]
+
+  books.each do |book|
+    puts book.title
+  end
+end
+
+def list_people
+  people = [Teacher.new('Pysics', 20, 'Harry Maguire', parent_permission: false),
+            Student.new('Pysics', 20, 'Jon Snow', parent_permission: false)]
+
+  people.each do |person|
+    puts person.name
+  end
+end
+
+def create_person(type)
+  case type
+  when 'teacher'
+    Teacher.new('Pysics', 20, 'Jon Snow', parent_permission: false)
+  when 'student'
+    Student.new('Class1', 19, 'Trent', parent_permission: false)
+  end
+end
+
+def create_book
+  Book.new('Harry Potter', 'JK Rowling')
+end
+
+def create_rental(date, book, person)
+  Rental.new(date, book, person)
+end
+
+def list_rentals(_person_id)
+  list_books
+  list_people
+  create_person('teacher')
+  create_book
+  create_rental('10/10/10', create_book, create_person('student'))
+end
```

Comment:
- [ ] These methods do not work as expected too. Kindly create a rental using the book and the people created by a user and a date then push it to the @rentals instance variable. Listing rentals should loop the @rentals instance variable to display the rentals created by that user.